frontend/src/pages/Results.tsx

```diff
@@ -48,23 +27,18 @@ const Results: React.FC<ResultsProps> = ({ simulationResults }) => {
                 </Tabs.List>
                 <Tabs.Panels>
                     <Tabs.Panel>
-                        <ResultConcPlot simulationResults={simulationResults ? simulationResults : fetchedResults!} />
+                        <ResultConcPlot simulationResults={simulationResults} />
                         <ResultConcTable
-                            initFinalDiff={
-                                simulationResults
-                                    ? simulationResults.results.initfinaldiff
-                                    : fetchedResults!.results.initfinaldiff
-                            }
+                            initialConcentrations={simulationResults.initialConcentrations}
+                            finalConcentrations={simulationResults.finalConcentrations}
                         />
                     </Tabs.Panel>
                     <Tabs.Panel>
-                        <Reactions simulationResults={simulationResults ? simulationResults : fetchedResults!} />
+                        {/* <Reactions simulationResults={simulationResults.initial_concentrations, simulationResults.final_concentrations} /> */}
```

Comment:
I think we can remove this reactions panel altogether in this PR. We want to make this Panel generation more general, and depending on the number of those `JSON | test` entries that is included..